src/Core/Content/Product/SalesChannel/Listing/ProductListingRoute.php

```diff
@@ -166,4 +166,41 @@ protected function doSearch(Criteria $criteria, SalesChannelContext $context): E
 
         return $result;
     }
+
+    protected function isRouteSupported(Request $request): bool
+    {
+        // Findologic should never trigger on home page, even if there are categories that would allow it.
+        if ($this->isHomePage($request)) {
+            return false;
+        }
+
+        // In case requests come from the home page, Findologic should not trigger on those.
+        if ($this->isRequestFromHomePage($request)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    protected function isHomePage(Request $request): bool
+    {
+        return $request->getPathInfo() === '/';
+    }
+
+    protected function isRequestFromHomePage(Request $request): bool
+    {
+        if (!$request->isXmlHttpRequest()) {
```

Comment:
When does this happen?

Reply:
When a filter on the homepage is selected for example.